http/src/request/channel/message/create_message.rs

```diff
@@ -368,6 +375,27 @@ impl<'a> CreateMessage<'a> {
         self
     }
 
+    /// Add multiple stickers to the message.
+    ///
+    /// Up to 3 stickers can be added to a message.  Stickers are Discord snowflakes.
+    ///
+    /// # Errors
+    ///
+    /// Returns a [`CreateMessageErrorType::TooManyStickers`] error type if more than three
+    /// stickers are provided
+    pub fn stickers(mut self, stickers: &'a [StickerId]) -> Result<Self, CreateMessageError> {
+        if stickers.len() > 3 {
```

Comment:
You should create a `validate_inner::stickers_limit` method to validate stickers count. See other methods of the module for examples.